### PR TITLE
Kanvi/Move the registrations for ops to a separate location

### PR DESCRIPTION
**Pull request by kanvi-nervana**

### CODE REVIEW

File: src/ops/ngraph_tracked_variable.cc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    .Output("ref: Ref(dtype)")
+    .Attr("shape: shape")
+    .Attr("dtype: type")
+    .Attr("just_looking: bool = false")
+    .Attr("container: string = ''")
+    .Attr("shared_name: string = ''")
+    .SetIsStateful()
+    .SetShapeFn(shape_inference::ExplicitShape);
+
+}  // namespace ngraph_bridge
```
Comment (sayantan-nervana) [inline src/ops/ngraph_tracked_variable.cc]: Another option is: we could have a single file for all ngraph ops. For example in TF, all nn ops are in a single file etc.

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -40,11 +40,13 @@ set(SRC
    ngraph_cluster_manager.cc
    ngraph_deassign_clusters.cc
    ngraph_encapsulate_clusters.cc
+   ops/ngraph_encapsulate_op.cc
    ngraph_encapsulate_op.cc
    ngraph_freshness_tracker.cc
    ngraph_mark_for_clustering.cc
    ngraph_rewrite_for_tracking.cc
    ngraph_rewrite_pass.cc
+   ops/ngraph_tracked_variable.cc
```
Comment (sayantan-nervana) [inline src/CMakeLists.txt]: Also do we need to update this as well? https://github.com/tensorflow/ngraph-bridge/blob/master/bazel/BUILD#L18

File: src/ops/ngraph_tracked_variable.cc (hunk 1 above)
Comment (kanvi-nervana) [reply src/ops/ngraph_tracked_variable.cc]: Yes, we can have something like ngraph_ops.cc and move registrations there.

File: src/enable_variable_ops/ngraph_tracked_variable.cc [hunk 3]
```diff
@@ -283,19 +283,6 @@ void NGraphVariableOp::Compute(OpKernelContext* ctx) {
...
-    .Attr("dtype: type")
-    .Attr("just_looking: bool = false")
-    .Attr("is_tf_just_looking: bool = false")
-    .Attr("copy_to_tf: bool = false")
-    .Attr("container: string = ''")
-    .Attr("shared_name: string = ''")
-    .Attr("ngraph_graph_id: int")
-    .SetIsStateful()
-    .SetShapeFn(shape_inference::ExplicitShape);
-
```
Comment (sayantan-nervana) [inline src/enable_variable_ops/ngraph_tracked_variable.cc]: we could remove `#include "tensorflow/core/framework/op_kernel.h"` from this file.

and maybe in other places too wherever applicable.

File: src/CMakeLists.txt [hunk 4]
```diff
@@ -65,6 +66,9 @@ if(NGRAPH_TF_ENABLE_VARIABLES_AND_OPTIMIZERS)
     list(REMOVE_ITEM SRC ngraph_rewrite_pass.cc)
     list(APPEND SRC enable_variable_ops/ngraph_rewrite_pass.cc)
 
+    list(REMOVE_ITEM SRC ops/ngraph_ops.cc)
+    list(APPEND SRC enable_variable_ops/ops/ngraph_ops.cc)
```
Comment (shresthamalik) [inline src/CMakeLists.txt]: We have some registerations here too, if you want to take care of them too
https://github.com/tensorflow/ngraph-bridge/blob/master/src/enable_variable_ops/ngraph_variable_modifiers.cc

File: src/CMakeLists.txt (hunk 2 above)
Comment (avijit-nervana) [reply src/CMakeLists.txt]: Yes

**Pull request merged**